docs/release_notes.rst

```diff
@@ -22,6 +22,8 @@ Bug Fixes
 
 * Fixed a false-positive recognition of section names causing D405 to be
   reported (#311, #317).
+* Fixed using the module filepath (instead of just the basename) to determine
+  its privateyness causing D100 to be reported incorrectly sometimes. (#326).
 
```

Comment:
The release notes should generally describe the changed behavior, not the implementation. This can be e.g., "Fixed an issue where module publicity was incorrectly determined by their top-level package" or something similar.

Also, please also include the issue number along with the PR number.